Startup notification

PabloAceG opened the pull request:
Create telegram notification to display information when the server starts, as well as a daily notification.

PabloAceG committed:
Notify telegram channel when server starts running

ZenTauro committed:
Add markup to messages

ZenTauro committed:
Format telegram messages better

PabloAceG committed:
Include statistics for disks, vms and format message

PabloAceG committed:
Merge branch 'main' into startup-notification

PabloAceG committed:
Reformat notification using roles

PabloAceG committed:
Use template to produce dynamic script

PabloAceG committed:
Minor changes

- Fix jinja detecting script variables.
- Change names for script as well as systemd service and timer.
- Delete Docker provision unused path.

PabloAceG committed:
Boot after boot and daily

ZenTauro commented on roles/notifications/files/notification-startup.service:
```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Notify Telegram channel, server has started running
+
+[Service]
+ExecStart=/usr/local/bin/notification-startup.sh
```
Shouldn't it be better to use an `After=${some_service}` directive instead of using a timer? Also, since this is a service that runs and then dies, I think it should use the `Type=oneshot`

PabloAceG replied:
It repeats daily and thus should be a timer.

ZenTauro commented on roles/notifications/files/notification-startup.service:
```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Notify Telegram channel, server has started running
```
```suggestion
Description=Notify Telegram channel, server has started running
Type=oneshot
```

PabloAceG replied:
Why would it need to use oneshot if the execution is controlled by a timer?

PabloAceG committed:
Fix #40

Notification did not display the disks for the Proxmox server, now
instead of `df` it is used pvesm, a native functionality provided by
Proxmox (as well as some other parsing steps).

_Consider this would only work for a pve server, in the future it might
be advisable to insert the command using template variables depending on
the type of host queried_.
